### PR TITLE
Alexlewis

### LINKED ISSUE #1: Initialise Phoenix

### LINKED ISSUE #3: Add book index
Decide if BookController will be the default controller or if we redirect to it

### LINKED ISSUE #4: Create database model for books

### LINKED ISSUE #5: Add form for creating new book
Posts data and creates database object

### LINKED ISSUE #6: Have index display all books

### PR DESCRIPTION
**Pull request by alextwilson**
Closes #1 
Closes #3 
Closes #4 
Closes #5 
Closes #6 

Adds Book schema and context

Add CRUD routes, views and controller actions
Context gives functions for database actions

### CODE REVIEW

**Comment by lewmoore** [comment]
:shipit:

**Comment by MissDove** [comment]
:shipit:

File: lib/pick_a_book/books/book.ex [hunk 1]
```diff
@@ -0,0 +1,21 @@
+defmodule PickABook.Books.Book do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+
+  schema "books" do
+    field :author, :string
+    field :description, :string
+    field :genre, :string
+    field :title, :string
```
Comment (bpourian) [inline lib/pick_a_book/books/book.ex:10]: description field should be text

File: lib/pick_a_book_web/router.ex [hunk 2]
```diff
@@ -17,9 +17,15 @@ defmodule PickABookWeb.Router do
     pipe_through :browser # Use the default browser stack
 
     get "/", PageController, :index
-    get "/books/new", BookController, :new
+    resources "/books", BookController
   end
 
+  # scope "/books", BookController do
```
Comment (bpourian) [inline lib/pick_a_book_web/router.ex]: Delete comments

File: priv/repo/migrations/20180306141115_create_books.exs [hunk 3]
```diff
@@ -0,0 +1,15 @@
+defmodule PickABook.Repo.Migrations.CreateBooks do
+  use Ecto.Migration
+
+  def change do
+    create table(:books) do
+      add :title, :string
+      add :author, :string
+      add :genre, :string
+      add :description, :string
```
Comment (bpourian) [inline priv/repo/migrations/20180306141115_create_books.exs]: Does description need to be a text?

**Review by bpourian — CHANGES_REQUESTED** [review]
Please see the comments

File: lib/pick_a_book_web/router.ex (hunk 2 above)
Comment (alextwilson) [reply lib/pick_a_book_web/router.ex]: Yes

**Comment by GiadaSimonetti** [comment]
🚀

**Comment by AlexandraGF** [comment]
🐹

**Review by bpourian — APPROVED** [review]

**Comment by bpourian** [comment]
🦑

**Pull request merged**